examples/kotlin-jvm/build.gradle.kts

```diff
@@ -25,16 +23,8 @@ dependencies {
     implementation(project(":kotlinx-benchmark-runtime"))
 }
 
-tasks.withType<JavaCompile> {
-    sourceCompatibility = "1.8"
-    targetCompatibility = "1.8"
-}
-
-
-tasks.withType<KotlinCompile> {
-    kotlinOptions {
-        jvmTarget = "1.8"
-    }
+kotlin {
+    jvmToolchain(8)
 }
```

Comment:
Maybe instead of using Java 8, we could use Java 21 to build and set the release flag to make the JVM classes target Java 8?

Also, use a Java 8 to launch and run the tests.

Reply:
> Maybe instead of using Java 8, we could use Java 21 to build and set the release flag to make the JVM classes target Java 8?

Do you mean in the example project?

Reply:
> Also, use a Java 8 to launch and run the tests.

Done, PTAL